google-cloud-firestore/src/test/java/com/google/cloud/firestore/BulkWriterTest.java

```diff
@@ -42,7 +42,7 @@
 import com.google.firestore.v1.BatchWriteResponse;
 import com.google.firestore.v1.Value;
 import com.google.firestore.v1.Write;
-import com.google.protobuf.GeneratedMessageV3;
+import com.google.protobuf.AbstractMessage;
```

Comment:
Can you use the `Message` interface instead?

Reply:
Updated in https://github.com/googleapis/java-firestore/pull/1709